open_humans/migrations/0010_grantproject_grant_date.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.9 on 2018-03-14 21:32
+from __future__ import unicode_literals
```

Comment:
Since this hasn't hit the production database yet, I'd prefer to have just one migration in the final merge (i.e. delete the two migrations and run `makemigrations` again).